defaults.yml

```diff
@@ -134,9 +137,9 @@ drupal:
         #   $> gunzip -c FILENAME.sql.gz | drush sqlc
         #
         # Command to extract text contents of the backup file.
-        contents_command: gunzip -c
+        contents_command: gzip -dc
```

Comment:
When we talked earlier on the DevOps call, it sounded like this change wasn't necessary to improve performance (or was it)?